config/routes.rb

```diff
@@ -64,6 +64,11 @@
 
     resource :report_schools, path: "report-schools", only: [] do
       get "start", action: :start
+
+      resource :confirm_schools, only: %i[show update], path: "confirm" do
+        get :start
+        post :remove
+      end
```

Comment:
I am not convinced we want a separate controller here, but I'm happy for that to get sorted when we have more parts of the flow.

Reply:
Yeah, I like splitting everything into a smaller chunks whenever possible. Please challenge me on this a lot, otherwise we'll end up with 3-line classes everywhere :P

In this case I went for a separate controller as this confirmation process is common for two separate user journeys (both csv and school search path). Another option would be to place it on `report_schools_controller`, but that did not exist at the time of writing this.

Reply:
I think small controllers are great, so long as the same session is not shared between multiple controllers - I shudder to think about that :D 
But there are no controllers sharing that same session yet, so... temporarily there is no problem ;)